### DISCUSSION TITLE
Welcome to vscode-go Discussions!

### DISCUSSION THREAD

**Discussion by hyangah** [golang/vscode-go#1118] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas - For concrete ideas, please [create new Feature Requests in the issue tracker](https://github.com/golang/vscode-go/issues/new?assignees=&labels=&template=feature_request.md&title=).
  * Engage with other community members.
  * Welcome others and remember that this is a community we build together 💪.. We follow the [Go Community Code of Conduct](https://golang.org/conduct). Remember that this is a community we
  build together 💪.

**Comment by dlipovetsky** [comment]
@hyangah Has some configuration changed since you posted this? I do not see a way to create a new discussion, yet the project README and the PR template both suggest using discussions.

According to GitHub:
> There is a control at the organization level which can limit Discussions so that only those with triage+ permissions can open new discussions. Anyone can still comment on an existing/opened discussion.
> -- https://github.com/github-community/community/discussions/29

----

I expect to see a "New Discussion" button, e.g.:
![image](https://user-images.githubusercontent.com/3445370/179309538-9e94dd44-0a4a-4b4a-9289-a640fd9eda2b.png)
(This is a screenshot of what I see when I open https://github.com/kovidgoyal/kitty/discussions. N.B. I have no special privileges in that repo, yet I can still create a new discussion).

This is what I see when I open https://github.com/golang/vscode-go/discussions:
![image](https://user-images.githubusercontent.com/3445370/179309883-4d0e43ec-9649-4174-b13d-69f12a3a3b3b.png)

Hopefully this reaches you. I'll file an issue if I don't see anything for a while.

**Reply by hyangah** [reply]
Thanks for the heads up @dlipovetsky . We adjusted our org-level permission and now the button should be back and available.
